weaviate/users/user_db_creator.go

```diff
@@ -0,0 +1,39 @@
+package users
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/connection"
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/except"
+	"github.com/weaviate/weaviate/client/users"
+)
+
+type UserDBCreator struct {
+	connection *connection.Connection
+
+	userID string
+}
+
+func (r *UserDBCreator) WithUserID(id string) *UserDBCreator {
+	r.userID = id
+	return r
+}
+
+func (r *UserDBCreator) Do(ctx context.Context) error {
+	payload := users.NewCreateUserParams().WithUserID(r.userID)
```

Comment:
should we add a runtime check that user_id is given? (same for the other endpoints that have required parameters)

Reply:
Seems we don't do it for (most?) of the other endpoints throughout the codebase, e.g. DELETE /schema/{class}

https://github.com/weaviate/weaviate-go-client/blob/9864f0c0d99ad07a9880c364cd76d12e61354c93/weaviate/schema/classDeleter.go#L25-L26

so there isn't a well-established pattern yet.
Is it worth introducing one for /users/... endpoints?

Reply:
I would have no issue adding parameter validation to all endpoints in a separate PR.